### DISCUSSION TITLE
Support Two Factor Authentication (2FA)

### DISCUSSION THREAD

**Discussion by dwightwatson** [laravel/nova-issues#4214] [Q&A] [answered]
Unless I'm missing something there doesn't appear to be any native support for 2FA. My application uses Jetstream with 2FA support which works great, but someone could easily visit `/nova/login` and authenticate there without 2FA tokens and have access to user information.

* Support hooking in the 2FA flow to secure back-end access, or
* Allow us to disable authentication directly with the Nova dashboard (i.e. users must authenticate through the front-end of the app).

**Comment by crynobone** [comment, marked as answer]
You can already utilise login via jetstream.

```php
config(['nova.routes.login' => '/login']);
```

**Reply by dwightwatson** [reply]
Great, thank you. Couldn't find this mentioned anywhere else.
